wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.0.0/13__add_signature_key.sql

```diff
@@ -0,0 +1,35 @@
+--
```

Comment:
Shouldn't this file be placed in `6.0.3` folder?

Reply:
:O

Reply:
I'll move scripts before merge

Reply:
I think it's good time to move it =)